nibabel/minc1.py

```diff
@@ -269,6 +269,7 @@ class MincHeader(Header):
     """
     # We don't use the data layout - this just in case we do later
     data_layout = 'C'
+    sizeof_hdr = 4
```

Comment:
I think I'll vote for reverting to `sniff_size`, unless there's a good reason to note the full header size.